### PR TITLE
Minor formatting changes

### PR DESCRIPTION
**Pull request by sriki18**
1. Add optional encoding for type 1 fonts as required by ProQuest.
2. Make page 1 details on professors' titles a little clearer.
3. In table of contents, make the word 'Appendix' appear for each appendix item.
4. Remove sections and subsections from appearing for the appendices in the table of contents page.

### CODE REVIEW

File: appendix.tex [hunk 1]
```diff
@@ -9,6 +9,9 @@
 \appendix
 \chapter{Appendix Title}
 
+% Appendix sections, subsections etc should not appear in the table of contents. So set tocdepth to 0 and set it back to 2 at the end. Do this for each appendix.
```
Comment (lotten) [inline appendix.tex]: Can I ask that you break/reflow this long line, to be consistent with what's already here?

File: preliminaries.tex [hunk 2]
```diff
@@ -12,12 +12,12 @@
 % Your name as it appears on official UCI records.
 \authorname{Your name}
 
-% Use the full name of each committee member.
+% Use the full name of each committee member and full title (e.g. Professor/Associate Professor).
```
Comment (lotten) [inline preliminaries.tex]: Similarly reflow here

File: thesis.tex [hunk 3]
```diff
@@ -18,13 +18,17 @@
...
 \usepackage{relsize}
+\usepackage[titletoc]{appendix}
 
 % Some other useful packages
 \usepackage{caption}
 \usepackage{subcaption}  % \begin{subfigure}...\end{subfigure} within figure
 \usepackage{multirow}
 \usepackage{tabularx}
 
+% Uncomment the following to attempt to enforce Type 1 or TrueType fonts. ProQuest does not want the type 3 fonts used by default as of Dec. 2019 - see https://support.proquest.com/articledetail?id=kA01W000000k9o2SAA . If you are unable to embed fonts such as 'Zapf Dingbats' or 'Symbol', try using raster images (.jpg or .png) instead of vector images (.pdf or .eps).
```
Comment (lotten) [inline thesis.tex]: And here as well, reflow/break long lines please

File: appendix.tex (hunk 1 above)
Comment (sriki18) [reply appendix.tex]: Will do this shortly and get back to you!

**Comment by sriki18** [comment]
I have made the changes in a follow up commit, let me know if that works!

**Comment by lotten** [comment]
Thanks a lot!

**Pull request merged**